src/scss/index.scss

```diff
@@ -1 +1,118 @@
+$primary-color: #253c6b;
```

Comment:
For this, you need a variable file to keep them and use them to extend bootstrap variables.

Reply:
See this lesson https://content.noroff.dev/css-frameworks/bootstrap-with-sass.html